README.md

```diff
@@ -113,6 +115,11 @@ You may also want to:
 
   1. Launch `amppkg` as a restricted user.
   2. Save its stdout to a rotated log somewhere.
+  3. Use the [provided tools](https://www.ampproject.org/docs/fundamentals/validate)
+     to verify that your published AMP documents are valid, for instance just
+     before publication, or with a regular audit of a sample of documents. The
+     [transforms](transformer/) are designed to work on valid AMP pages, and
```

Comment:
and *may* break.

The way this reads is that they are *designed* to break invalid AMP pages, which is not true.

Reply:
Oops, that used to be there, not sure what happened. Fixed.